examples/complete/README.md

```diff
@@ -0,0 +1,8 @@
+# End-to-end example
+
+An end-to-end complete example that will provision the following:
+- Create a new resource group if one is not passed in.
+- Create a new Cloud Internet Services (CIS) instance.
+- Add domain in CIS.
```

Comment:
Perhaps put the values that are used in this example. + some instructions on how to get the example work with real world values.